src/plugins/initiatives/createGraph.test.js

```diff
@@ -80,10 +80,11 @@ function exampleNodeAddress(id: number): NodeAddressT {
   return NodeAddress.fromParts(["example", String(id)]);
 }
 
-function discourseInitiativeAddress(id: number): NodeAddressT {
+function testInitiativeAddress(num: number): NodeAddressT {
   return NodeAddress.append(
     initiativeNodeType.prefix,
-    ...NodeAddress.toParts(topicAddress("https://example.com", id))
+    "TEST_SUBTYPE",
+    num.toString()
```

Comment:
this is a total nit so no change requested -- i personally like to use `String(x)` because it works more consistently across types ... consider that if `x == null` then `String(x)` is fine but `x.toString()` isn't.

Reply:
Agreed, generally better. Updated.
I got a bit complacent here because the flow signature was a non-nullable `number`.